8.0/Dockerfile

```diff
@@ -58,14 +70,14 @@ RUN apt-get update && apt-get install --no-install-recommends libldap2-dev -y \
 RUN docker-php-ext-configure pcntl --enable-pcntl \
     && docker-php-ext-install pcntl
 
-RUN pecl install xdebug-2.9.0
+RUN pecl install xdebug-3.0.2
 RUN echo "zend_extension=`php -i | grep ^extension_dir | cut -f 3 -d ' '`/xdebug.so" >> /usr/local/etc/php/conf.d/xdebug.ini
 
 RUN docker-php-ext-install soap
 
 ENV COMPOSER_HOME=/composer
-COPY --from=composer:1.10.17 /usr/bin/composer /usr/local/bin/composer1
-COPY --from=composer:2.0.7 /usr/bin/composer /usr/local/bin/composer2
+COPY --from=composer:1.10.19 /usr/bin/composer /usr/local/bin/composer1
```

Comment:
Can we keep the same versions like in the other branches and change it later across all versions?